src/checkstyle/checkstyle.xml

```diff
@@ -73,6 +68,13 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <!-- All Java AST specific tests live under TreeWalker module. -->
     <module name="TreeWalker">
 
+        <!-- Allow use of comment to suppress javadocstyle -->
```

Comment:
The `SuppressionCommentFilter` should go inside of the `TreeWalker` module.
See https://github.com/checkstyle/checkstyle/issues/4714